lib/screens/container_screen.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:basic_widgets/strings.dart';
+
+class ContainerScreen extends StatelessWidget {
+  const ContainerScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(title: const Text(Strings.container)),
+        body: Container(
+          color: Colors.black,
+          alignment: Alignment.center,
+          child: Container(
+            decoration: BoxDecoration(
+                color: Colors.green,
+                borderRadius: BorderRadius.circular(10),
+                border: Border.all(color: Colors.red, width: 2)),
+            //color: Colors.green,
```

Comment:
If you do not need something in the code - just remove it. Keeping something commented it is a bad practice.

Reply:
sorry, I forget about this comment. Delete it